### DISCUSSION TITLE
Is push notification in v2 ready ?

### DISCUSSION THREAD

**Discussion by vuletiki** [WalletConnect/walletconnect-monorepo#1245] [Developer Support]
I cloned monorepo few month ago and see some logic to push notification in legacy and push/relay server code.
But now server was moved, I wonder is it done ? or when it's done ?
Thanks

**Comment by timionius** [comment]
any update? does Push Notification https://docs.walletconnect.com/web3wallet/push-notifications work? 
My case is connect to dApp from the wallet. I tried to debug but there's no any messages coming during connection establishing dApps -> Wallet.
